variable/special_values.cpp

```diff
@@ -45,4 +45,8 @@ Variable neg_inf_to_num(const VariableConstView &var,
   return transform(var, replacement, element::negative_inf_to_num);
 }
 
+Variable isfinite(const VariableConstView &var) {
+  return transform(var, element::isfinite);
+}
```

Comment:
Why not generated any more?

Reply:
make_unary{isfinite} was causing compile errors as it attempted to set the bool return value to a vector. I didn't think the inplace version of this operator was needed so this seemed like the simplest solution. If the inplace version is needed I will need to take a step back and figure out how to get it to work.

Reply:
The codegen uses the `NO_OUT` arg, so the inplace version should not even be generated? 

I don't think we should just silently remove functions, so yes, we need to get to the bottom of this, understanding why this failed, despite apparently being disabled by `NO_OUT`?

Reply:
I think NO_OUT only affects the python function generation and not the C++ side. Adding something like the below line from python_unary.cpp to variable_unary.cpp may be the solution.

`if constexpr(std::is_same_v<T, Variable> && @GENERATE_OUT@)`

Reply:
Right. Can you try what you suggest?